tests/test_run_state_async.py

```
@@ -0,0 +1,90 @@
+        manager._cache.clear()
+        fetched_again, _ = await manager.get_run(run_id)
+        assert fetched_again["status"] == "running"
+
+    asyncio.run(_test())
+
+
+def test_list_runs(tmp_path):
+    manager = RunStateManager(runs_root=tmp_path)
+
+    # Create multiple runs (async)
+    async def create_runs():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `create_runs`

(ANN202)